tests/test_modeling_flax_common.py

```diff
@@ -152,7 +152,7 @@ def test_equivalence_flax_to_pt(self):
                 fx_outputs = fx_model(**prepared_inputs_dict)
                 self.assertEqual(len(fx_outputs), len(pt_outputs), "Output lengths differ between Flax and PyTorch")
                 for fx_output, pt_output in zip(fx_outputs, pt_outputs):
-                    self.assert_almost_equals(fx_output, pt_output.numpy(), 1e-3)
+                    self.assert_almost_equals(fx_output, pt_output.numpy(), 5e-3)
```

Comment:
I guess it's fine to have `5e-3` if ELECTRA needs it. However, the smaller the error tolerance, the better so if tests pass with `1e-3` than let's leave them at `1e-3` :-)

Reply:
@LysandreJik - If I remember correctly, ELECTRA had very high logit values which could explain the required higher tolerance here - is that correct?

Reply:
Okok, I'll check which ones actually need to be changed. Btw is there a reason for the explained different between Jax and Pytorch? if I understand correctly the weights are exactly the same

Reply:
It's rounding errors that can add up

Reply:
It has logits in the range of +-1e[0-1], so 5e-3 is definitely reasonable!